pkg/skuba/actions/node/upgrade/apply.go

```diff
@@ -48,6 +48,9 @@ func Apply(client clientset.Interface, target *deployments.Target) error {
 	}
 	currentVersion := currentClusterVersion.String()
 	latestVersion := kubernetes.LatestVersion().String()
+	if strings.Contains(target.Cache.OsRelease["VERSION_ID"], "15.1") && !strings.Contains(latestVersion, "1.17") {
+		fmt.Printf("Node OS migration/upgrade required first.\n\n")
```

Comment:
shouldn't we exit by issuing a Fatal instead?

Reply:
Fixed.